src/core/Akka.Streams/Dsl/Sink.cs

```diff
@@ -375,6 +375,7 @@ public static class Sink
         /// <param name="parallelism">TBD</param>
         /// <param name="action">TBD</param>
         /// <returns>TBD</returns>
+        [Obsolete("Use `ForEachAsync` instead, it allows you to choose how to run the procedure, by calling some other API returning a Task or using Task.Run. Obsolete since 1.5.2")]
```

Comment:
Instead of making this obsolete, why not make an overloaed version that takes a `Func<TIn, Task>` instead of an `Action<TIn>` instead?

Reply:
To eventually delete code instead of keep adding more: you can now do everything with `ForEachAsync`, hence why is deprecated. On top of that, we are keeping up date with Akka's public API.

Reply:
That makes sense - agree with @ismaelhamed

Reply:
OK, make sense